examples/style.xml

```diff
@@ -6,13 +6,13 @@
 <!ENTITY google_mercator "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over">
 ]>
 
-<Map srs="&google_mercator;">
+<Map>
```

Comment:
I don't think it is proj. Could it be a lxml issue? We definitely need more coverage.